content/simplified-chinese/python-basics/basics/comments.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "注释"
```

Comment:
For all these, can we also include what the equivalent English word is (since English is still a widely accepted universal language of programming, it's important the students know both)

```suggestion
title: "注释 (Comments)"
```